extensions/cornerstone/src/panels/PanelSegmentation.tsx

```diff
@@ -0,0 +1,286 @@
+import React from 'react';
+import { SegmentationTable } from '@ohif/ui-next';
+import { callInputDialog, colorPickerDialog, createReportAsync } from '@ohif/extension-default';
+import { useActiveViewportSegmentationRepresentations } from '../hooks/useActiveViewportSegmentationRepresentations';
+
+export default function PanelSegmentation({
+  servicesManager,
+  commandsManager,
+  extensionManager,
+  configuration,
+  children,
+}: withAppTypes) {
+  const { segmentationService, viewportGridService, uiDialogService, customizationService } =
+    servicesManager.services;
+
+  const { segmentationsWithRepresentations, disabled } =
+    useActiveViewportSegmentationRepresentations({
+      servicesManager,
+    });
+
+  const handlers = {
+    onSegmentationAdd: async () => {
+      segmentationService.createLabelmapForViewport(viewportGridService.getActiveViewportId());
+    },
+
+    onSegmentationClick: (segmentationId: string) => {
+      segmentationService.setActiveSegmentation(
+        viewportGridService.getActiveViewportId(),
+        segmentationId
+      );
+    },
+
+    onSegmentAdd: segmentationId => {
+      segmentationService.addSegment(segmentationId);
+    },
+
+    onSegmentClick: (segmentationId, segmentIndex) => {
+      segmentationService.setActiveSegment(segmentationId, segmentIndex);
+      segmentationService.jumpToSegmentCenter(segmentationId, segmentIndex);
+    },
+
+    onSegmentEdit: (segmentationId, segmentIndex) => {
+      const segmentation = segmentationService.getSegmentation(segmentationId);
```

Comment:
This bit should be in a commands module so it can be attached to a keyboard shortcut or the like.